bundles/azure-service-broker-0.0.1/chart/azure-service-broker/templates/post-install-job.yaml

```diff
@@ -59,3 +59,58 @@ spec:
                         echo "ServiesBroker is not ready. Timeout reached";
                         exit 1;
                       fi
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+    name: azure-service-broker-docu-checker
+    labels:
+        app: {{ template "fullname" . }}
+        chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+        release: {{ .Release.Name }}
+        heritage: {{ .Release.Service }}
+    annotations:
+        "helm.sh/hook": post-install,post-upgrade
```

Comment:
misleading, file is called **post-install-job.yaml**, but this is executed for install and upgrade. Another reason to have that logic in separate space and not mix that together